TypeScript.YAML-tmLanguage

```diff
@@ -169,7 +160,7 @@ repository:
       '5': { name: keyword.operator.ts }
       '6': { name: entity.name.function.ts }
       '7': { name: keyword.operator.ts }
-    end: '(?=\}|;|,|$)|(?<=\})'
```

Comment:
Hm - I guess there's not really a problem, since you'll just highlight as a block anyway. This should be fine